src/components/application_manager/src/application_impl.cc

```diff
@@ -399,6 +399,33 @@ bool ApplicationImpl::audio_streaming_allowed() const {
   return audio_streaming_allowed_;
 }
 
+bool ApplicationImpl::SetVideoConfig(protocol_handler::ServiceType service_type,
+                                     const smart_objects::SmartObject& params) {
+  using namespace protocol_handler;
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  if (ServiceType::kMobileNav == service_type) {
+    // See StartStreaming(). We issue SetVideoConfig and StartStream
+    // only when streaming is not approved yet
+    if (!video_streaming_approved()) {
+      LOG4CXX_TRACE(logger_, "Video streaming not approved");
+      MessageHelper::SendNaviSetVideoConfig(
+          app_id(), application_manager_, params);
+      return true;
+    }
+  }
+  return false;
+}
+
+void ApplicationImpl::OnNaviSetVideoConfigDone(
```

Comment:
Sure, fixed in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/11edb020834ecd2f52e2b9c1a16d64ff4dc99dcb.